manifest/schema2/manifest.go

```diff
@@ -132,3 +132,16 @@ func (m *DeserializedManifest) MarshalJSON() ([]byte, error) {
 func (m DeserializedManifest) Payload() (string, []byte, error) {
 	return m.MediaType, m.canonical, nil
 }
+
+// ConfigType represents the type of configuration inside the manifest based on
+// on the media type of the config. Return values are "image" for the default
```

Comment:
Haven't we called this class above? Also, shouldn't this classification be part of the policy mechanism, rather than the manifest subsystem?